tcat-app/inputparams.py

```diff
@@ -1,39 +1,57 @@
         ['input', 'number', 'n_sats_per_plane', 'Number of satellite per orbital plane', 2, [1, 1000]],
         ['input', 'number', 'plane_distribution_angle', 'Distribution angle of orbital planes (deg)', 180, [180, 360]],
     ],
-    ('launch_vehicle_configuration', 'Launch Vehicle Configuration'): [
-        ['select', 'text', 'launcher', 'Launch vehicle name and variant', 'Soyuz_2.1a_Fregat',
-         ['Ariane_62', 'Ariane_64', 'Soyuz_2.1a_Fregat', 'Soyuz_2.1b_Fregat', 'Soyuz']],
+    ('launcher', 'Launcher'): [
+        ['checkbox', '', 'launcher_use_database', 'Use database', True, [True, False]],
+        ['select', 'text', 'launcher', 'Launch vehicle name and variant', 'Soyuz_2.1a_Fregat', ['Ariane_62', 'Ariane_64', 'Soyuz_2.1a_Fregat', 'Soyuz_2.1b_Fregat', 'Soyuz']],
         ['select', 'text', 'launch_site', 'Launch Site', 'Baikonur', ['Korou', 'Baikonur']],
         ['select', 'text', 'orbit_type', 'Orbit Type', 'LEO', ['LEO', 'SSO', 'LPEO', 'Polar', 'MTO', 'GTO', 'GTO+']],
         ['input', 'number', 'dispenser_tech_level', 'Dispenser technological level', 1.1, [0.1, 1.9]],
-        ['input', 'text', 'custom_launcher_name', 'Custom launch vehicle name', None, '^[A-Za-z0-9 ]+$'],
-        ['input', 'number', 'custom_launcher_performance', 'Custom launch vehicle performance (kg)', None,
-         [0.1, 99999.9]],
-        ['input', 'number', 'fairing_diameter', 'Fairing diameter (m)', None, [0.1, 100.0]],
-        ['input', 'number', 'fairing_cylinder_height', 'Fairing height of the cylindrical section (m)', None,
-         [0.1, 100.0]],
-        ['input', 'number', 'fairing_total_height', 'Fairing total height (m)', None, [0.1, 100.0]],
-        ['select', 'text', 'interpolation_method', 'Interpolation method for the launch vehicle performance', 'linear',
-         ['linear', 'nearest', 'cubic', 'nearest-up', 'zero', 'slinear', 'quadratic', 'previous', 'next']]
+        ['input', 'text', 'launcher_name', 'Custom name', None, '^[A-Za-z0-9 ]+$'],
+        ['input', 'number', 'launcher_performance', 'Custom performance (kg)', None, [0.1, 99999.9]],
+        ['input', 'number', 'launcher_fairing_diameter', 'Fairing diameter (m)', None, [0.1, 100.0]],
+        ['input', 'number', 'launcher_fairing_cylinder_height', 'Fairing height of the cylindrical section (m)', None, [0.1, 100.0]],
+        ['input', 'number', 'launcher_fairing_total_height', 'Fairing total height (m)', None, [0.1, 100.0]],
+        ['select', 'text', 'launcher_perf_interpolation_method', 'Interpolation method for the launch vehicle performance', 'linear', ['linear', 'nearest', 'cubic', 'nearest-up', 'zero', 'slinear', 'quadratic', 'previous', 'next']]
     ],
-    ('orbits_configuration', 'Orbits Configuration'): [
+    ('kickstage', 'Kickstage'): [
+        ['checkbox', '', 'kickstage_use_database', 'Use database', True, [True, False]],
+        ['input', 'text', 'kickstage_name', 'Name', 'My Kickstage', '^[A-Za-z0-9 ]+$'],
```

Comment:
just like for launchers, need here a drop down list with the kick stages existing in the database : [Fregat, Fregat_M, AVUM_plus]